src/components/DataTable/Cells/holidays.js

```diff
@@ -55,17 +55,24 @@ const requestedDate = {
   id: 'requestedDate',
   Header: 'Requested',
   accessor: holiday => {
+    const { dateCreated, eventStatus: { eventStatusId } } = holiday;
     const today = moment();
-    const diff = today.diff(holiday.dateCreated, 'days');
-    if (
-      diff >= 5 &&
-      holiday.eventStatus.eventStatusId === holidayStatus.PENDING
-    ) {
+    const totalDaysSinceRequest = today.diff(dateCreated, 'days');
+
+    if (totalDaysSinceRequest === 0) {
+      return 'Today';
+    }
+    else if (totalDaysSinceRequest === 1) {
+      return '1 Day Ago';
+    } 
+    else if (totalDaysSinceRequest >= 5 && eventStatusId === holidayStatus.PENDING) {
       return (
-        <span style={{ color: 'red', fontWeight: 600 }}>{diff} Days Ago</span>
+        <span style={{ color: 'red', fontWeight: 600 }}>{totalDaysSinceRequest} Days</span>
       );
+    } else {
+      return `${totalDaysSinceRequest} Days Ago`;
```

Comment:
Check out moment's `.fromNow()` call. It might strip away almost all of this logic.

Reply:
Nice... Ok, so i've updated it. We're not controlling the terminology now but I think that's ok. Breakdown is seconds/an hour/hours/a day/days/a week/weeks ago etc.

Reply:
It's used elsewhere in the app and works well, so it'll keep things consistent. Nice work, looks much cleaner.